com.github.gyunaev.spivak.json

```diff
@@ -0,0 +1,99 @@
+{
+    "app-id": "com.github.gyunaev.spivak",
+    "command": "spivak",
+    "rename-desktop-file": "spivak.desktop",
+    "rename-icon": "spivak",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "finish-args":[
+	"--socket=x11",
```

Comment:
if it's socket x11 you should have --share=ipc as well. Also, all Qt apps support wayland these days (although do test with something this 'artisan'